vita3k/modules/SceAudio/SceAudio.cpp

```diff
@@ -164,8 +164,16 @@ EXPORT(int, sceAudioOutSetPortVolume_forUser) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAudioOutSetVolume) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceAudioOutSetVolume, int port, int channelFlags, int *vol) {
+    // 32768 is max volume
+    int left = *vol;
```

Comment:
Also check channelFlags, there might be instances where only one channel is to be set

Reply:
```suggestion
    const int left = vol[0];
```